### PR TITLE
feat(skip): option to skip package manager selection

### PR DESCRIPTION
**Pull request by qxb3**

### CODE REVIEW

**Comment by EvolutionX-10** [comment]
There's a problem, it only prompts for manager if a user has both, yarn and npm
Else it will just continue installing with what the user has
Here are the [lines](https://github.com/sern-handler/cli/blob/main/src/commands/init.js#L69-L78) that indicate the logic
If you got any ideas to fix it, feel free to do

**Comment by qxb3** [comment]
@EvolutionX-10 can you try it now?

**Comment by Allyedge** [comment]
I suggest waiting until #19 is merged. So that there won't be any conflicts.

I will review the PR after that.

File: src/prompts/init.js [hunk 1]
```diff
@@ -76,9 +76,21 @@ export const which_manager = {
...
+			description: 'Skip selection',
+			value: 'skip',
+		},
 	],
 };
 
+export const skip_install_dep = {
+	name: 'skip_install_dep',
+	type: 'confirm',
+	message: `Do you want ${blueBright('me')} to install dependecies?`,
```
Comment (EvolutionX-10) [inline src/prompts/init.js:90]:
```suggestion
	message: `Do you want ${blueBright('me')} to install dependencies?`,
```
Typo

**Review by EvolutionX-10 — CHANGES_REQUESTED** [review]
I'll test them soon

**Review by Allyedge — DISMISSED** [review]
Everything looks fine, good job.

**Review by EvolutionX-10 — APPROVED** [review]
Nice @qxb3

**Pull request merged**